themes-default/slim/views/config_search.mako

```diff
@@ -18,8 +18,6 @@ const startVue = () => {
         metaInfo: {
             title: 'Config - Episode Search'
         },
-        computed: Object.assign(store.mapState([]), {
-        }),
```

Comment:
You're still pulling the config on this file: https://github.com/pymedusa/Medusa/blob/520807626e929bd657c38f5892908d95359ce609/themes-default/slim/views/config_search.mako#L206-L207